src/main/java/com/littleBeasts/GameLogic.java

```diff
@@ -127,18 +132,27 @@ public static void setState(GameState state) {
 
     private static void triggerBattle() {
         int x = 0;
+        boolean left = false;
```

Comment:
If I understood the UC for this var correctly I would rather call it faceLeft or something like that. Left is a bit too general here I think

Reply:
good Idea